### PR TITLE
implemented task

### PR DESCRIPTION
**Pull request by Olegator191**

### CODE REVIEW

File: src/main/java/core/basesyntax/Figure.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw, AreaCounter {
+    private String color;
```
Comment (KostyaNikiforov) [inline src/main/java/core/basesyntax/Figure.java:4]: Can u use the enum here? It is better to avoid primitives if it is possible

**Review by KostyaNikiforov — CHANGES_REQUESTED** [review]
Good solution!:fire:

File: src/main/java/core/basesyntax/Figure.java (hunk 1 above)
Comment (Olegator191) [reply src/main/java/core/basesyntax/Figure.java]: Yes,it`s done!)

File: src/main/java/core/basesyntax/Figure.java [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 public abstract class Figure implements Draw, AreaCounter {
-    private String color;
+    private Color color;
 
     public Figure(String color) {
-        this.color = color;
+        this.color = Color.valueOf(color);
     }
 
     public String getColor() {
```
Comment (KostyaNikiforov) [inline src/main/java/core/basesyntax/Figure.java:10]: Why don't you use Enum here too?

**Review by KostyaNikiforov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**